### DISCUSSION TITLE
Generate DB schemas based on rust codes.

### DISCUSSION THREAD

**Discussion by dhra** [SeaQL/sea-orm#329] [Q&A]
How is it possible to create migration via sea-orm based on a schema file in rust? Is there any documentation to guide newcomers step by step (I checked the main docs by sea-orm). Something like diesel docs with step by step guide would be more useful.

**Comment by billy1624** [comment]
Hi @dhra, welcome! Not really a migration but you can create db tables via Rust code. https://www.sea-ql.org/SeaORM/docs/generate-database-schema/create-table

**Comment by dhra** [comment]
Thanks @billy1624, Is there any plan for migrations via rust codes in the SeaQL?

**Reply by billy1624** [reply]
We planed to introduce a schema manager that include migration feature on our next minor version.

**Comment by dhra** [comment]
Is it possible to integrate `https://crates.io/crates/models` with the sea-orm? @billy1624
